.pre-commit-config.yaml

```diff
@@ -0,0 +1,22 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+# FIXME: include these directories in the commit hook, some day...
+exclude: '^(vpc-template|docs)/'
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v2.4.0
+  hooks:
+    - id: trailing-whitespace
+    - id: end-of-file-fixer
+    - id: check-yaml
+    - id: check-added-large-files
+    - id: check-json
+    - id: flake8
```

Comment:
It seems pre-commit says to not use this hook anymore, and to use flake8's repo instead (see the "use gitlab.com/pycqa/flake8 instead" note on https://pre-commit.com/hooks.html)

Reply:
Good find! Thanks. It appears that somehow fixes the fighting over file formatting: